### ISSUE TITLE
Accessibility swarm on the Brand component

### ISSUE THREAD

**Issue by ghost** [BBC-archive/psammead#561]
Blocked on #607

**Is your feature request related to a problem? Please describe.**
Carry out a two `devs` accessibility swarm for the `Brand` component.

**Describe the solution you'd like**
Go through the [accessibility checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and confirm everything is working as expected and fix any issues that might crop up.

**Describe alternatives you've considered**
No alternative for accessibility 😁 

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by greenc05** [comment]
Lets make sure we have met the following accessibility acceptance criteria before carrying out this swarm:

Given I use a screen reader
When I navigate to the banner landmark on Pidgin
Then BBC News Pidgin brand logo is the first thing to be announced
And brand logo will be announced as 'BBC News, Pidgin’
**And ‘BBC News’ is announced with an English accent**

**Comment by ghost** [comment]
On FireFox high contrast mode the brand SVG is displayed as white on a white background and therefore can't be seen. https://github.com/bbc/psammead/issues/736

**Comment by dr3** [comment]
(edit me)

❓ = Not done
✅ = Done, pass
❌ = Done, fail

❓ ZoomText Magnifier/Reader Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Magnifier with Screen Reader capabilities)

❓ Dragon Naturallyspeaking Version 13
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Speech Recognition)

❓ JAWS Version 17
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Reader)

❓ Read&Write Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Reading Solution)

❌ VoiceOver iOS (iPhone/iPad) Latest Version
With Safari (Latest Version) on iOS (Latest Version)
(Screen Reader)
Raised https://github.com/bbc/psammead/issues/743

❓ NVDA (Latest Version)
With Firefox (Latest Version) on Windows (XP/Vista/7/8/10)
(Screen Reader)

❓VoiceOver OS (Mac) (Latest Version)
With Safari (Latest Version) on Mac OS (Latest Version)
(Screen Reader)

❓TalkBack (Latest Version)
With Chrome (Latest Version) on Android (Latest Version)
(Screen Reader)

**Comment by greenc05** [comment]
Please don't forget VoiceOver Mac and Talkback

**Comment by j-pendlebury** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail

✅ ZoomText Magnifier/Reader Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Magnifier with Screen Reader capabilities)

❓ Dragon Naturallyspeaking Version 13
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Speech Recognition)

✅ JAWS Version 17
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Screen Reader)

✅ Read&Write Latest Version
With Internet Explorer 11 on Windows (XP/Vista/7/8/10)
(Reading Solution)

✅ VoiceOver iOS (iPhone/iPad) Latest Version
With Safari (Latest Version) on iOS (Latest Version)
(Screen Reader)
Raised #743 - now fixed

✅ NVDA (Latest Version)
With Firefox (Latest Version) on Windows (XP/Vista/7/8/10)
(Screen Reader)

✅ VoiceOver OS (Mac) (Latest Version)
With Safari (Latest Version) on Mac OS (Latest Version)
(Screen Reader)

✅ TalkBack (Latest Version)
With Chrome (Latest Version) on Android (Latest Version)
(Screen Reader)

✅ Manual checks

**Comment by j-pendlebury** [comment]
Known issue with SVG not showing on Firefox high contrast mode: https://github.com/bbc/psammead/issues/736

**Comment by dr3** [comment]
Brand size issue on zoom https://github.com/bbc/simorgh/issues/1905
